.rubocop.yml

```diff
@@ -29,6 +29,7 @@ Sorbet/TrueSigil:
   Exclude:
     - "**/*.rake"
     - "lib/**/*.rb"
+    - "test/expectations/formatting/*.rb"
```

Comment:
Let's exclude the entire expectations folder. There's nothing there that we need to run RuboCop over.